dask_cloudprovider/cloudprovider.yaml

```diff
@@ -4,9 +4,11 @@ cloudprovider:
     fargate_workers: False # Use fargate mode for the workers
     scheduler_cpu: 1024 # Millicpu (1024ths of a CPU core)
     scheduler_mem: 4096 # Memory in MB
+#   scheduler_extra_args: "--tls-cert,/path/to/cert.pem,--tls-key,/path/to/cert.key,--tls-ca-file,/path/to/ca.key"
```

Comment:
Glad to see the example here.
Is there anything gained, do you think, by also supporting whitespace-delimited (for copy&paste from the command line) or lists (for copy&paste from code)?

Reply:
I like the whitespace-delimited idea. That would definitely be convenient for copy/paste from actual command lines. Although for generic command line arguments (that could have quoted arguments with spaces) it could get tricky to parse, e.g.

worker_extra_args: "--name \"Joe Worker Type\""

Based on that I lean toward punting on support for whitespace delimiting, but let me know what you think. If you feel strongly about it, I don't mind taking a crack at parsing for it and see how it goes. (Have to be aware of quoted arguments and avoid splitting on those, etc.)

Reply:
If you support a list here, then the user can be explicit. I don't recommend trying to be clever here. Perhaps there is a similar situation elsewhere in the config? 
To be sure, I am OK to accept the PR as it is, until someone were to ask for something more comprehensive.

Reply:
I did a quick review and all other config values are loaded directly, without any additional parsing, e.g. `self.image = self.config.get("image")`

I like the idea to wrap up this PR and then revisit if someone asks for something more comprehensive. Thanks!